app/models/pipeline.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Pipeline < ApplicationRecord
+  has_many :harvest_definitions
```

Comment:
Why is it `has_many`? A pipeline can have multiple harvest definitions? Is it not one harvest and many enrichments?

Reply:
Yeah we are using the same model for enrichments and harvests. So a harvest_definition could either be an enrichment or a harvest. 

The idea though is 1 harvest and many enrichments.